css/components/cards.css

```diff
@@ -0,0 +1,82 @@
+.card{
+    display: flex;
+    padding: var(--space-sm);
+    border-radius: var(--radius-md);
+    color: var(--background-night-pastel);
+    gap: var(--space-sm);
+    position: relative;
+}
+
+.vertical-card{
+    flex-direction: column;
+    max-width: 18rem;
+}
+
+.horizontal-card{
+    display: flex;
+    max-width: 40rem;
+}
+
+.horizontal-card>.card-image{
+    max-width: 200px;
+}
+
+.card-image{
+    background-color: var(--background-night-pastel);
```

Comment:
you can use position relative for img and then use text overlay relative to img.